doc/neps/nep-template.rst

```diff
@@ -13,15 +13,20 @@ Abstract
 
 The abstract should be a short description of what the NEP will achieve.
 
-
-Detailed description
---------------------
+Motivation
+----------
 
 This section describes the need for the NEP.  It should describe the existing
-problem that it is trying to solve and why this NEP makes the situation better.
-It should include examples of how the new functionality would be used and
-perhaps some use cases.
+problem, who it affects, what it is trying to solve, and why.
```

Comment:
After this, I suggest: "This section should explicitly address the scope and requirements of the proposed change."